src/lib/auth.js

```diff
@@ -296,30 +296,35 @@ class Auth {
 
   /**
    * JWT middleware for WebUI
-   * @return {Function}
    */
   jwtMiddleware() {
     return (req: $RequestExtend, res: $Response, _next: NextFunction) => {
-      if (req.remote_user !== null && req.remote_user.name !== undefined) return _next();
+      if (req.remote_user !== null && req.remote_user.name !== undefined) {
+       return _next();
+      }
 
       req.pause();
       const next = function(_err) {
         req.resume();
         return _next();
       };
 
-      req.remote_user = buildAnonymousUser();
-
-      let token = (req.headers.authorization || '').replace('Bearer ', '');
-      if (!token) return next();
+      const token = (req.headers.authorization || '').replace('Bearer ', '');
+      if (!token) {
+        return next();
+      }
 
       let decoded;
       try {
         decoded = this.decode_token(token);
-      } catch (err) {/**/}
+      } catch (err) {
+       // FIXME: intended behaviour, do we want it?
```

Comment:
I'm not sure what we want to do here. @Meeeeow @ayusharma

Reply:
I think this `try {..} catch() {..}` can be removed. It will not invoked and useless. 
- we have `if(decoded)` in the upcoming lines. 
- Also, there is a `try {..} catch() {..}` in `decode_token` method which will throw error in any catch situation.

Reply:
But the catch trow `throw Error[401](err.message);` so, we need the `catch`

Reply:
Yes. You are right. I just checked it by creating a dummy code. 😀

Reply:
:) so .. give me your 👍 to be able to merge if you are happy